code/modules/screen_alert/command_alert.dm

```diff
@@ -77,9 +77,12 @@
 		return
 	for(var/mob/faction_receiver in alert_receivers)
 		if(faction_receiver.faction == human_owner.faction || isdead(faction_receiver))
-			faction_receiver.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>COMMAND ANNOUNCEMENT:</u></span><br>" + text, /atom/movable/screen/text/screen_text/command_order)
+			var/faction_title = "Command" // the default title, for tgmc announcements
+			if(human_owner.faction != FACTION_TERRAGOV)
+				faction_title = "[GLOB.faction_to_acronym[human_owner.faction]] Command" // if the sender doesn't belong to the TGMC, clarify the faction in the title
```

Comment:
I wouldn't bother with the exclusion for tgmc tbh.

Reply:
TGMC exclusion is an antique from when I worked on 14957, I want send orders from TGMC leaders to look like a CIC announcement. I'll change this if you really want though

Reply:
its just consistant